### PR TITLE
Report details tabs

### PR DESCRIPTION
**Pull request by adi130987**

### CODE REVIEW

**Comment by adi130987** [comment]
connects unicef/etools-issues#846

File: pmp/app-elements/etools-status/etools-action-button.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
       text-align: center;
       font-weight: 500;
       line-height: 34px;
-      height: 100%;
+      /*height: 100%;*/
```
Comment (daniel-tabarcea) [inline pmp/app-elements/etools-status/etools-action-button.html]: shouldn't we remove this altogether instead of commenting it?

Comment (adi130987) [reply pmp/app-elements/etools-status/etools-action-button.html]: done :)

File: pmp/app-elements/reports/report-progress.html [hunk 2]
```diff
@@ -252,167 +167,49 @@
...
-                    }],
-                    "display_type": "percentage",
-                    "total": {
-                        "c": 0.09371356828983948,
-                        "d": 11151,
-                        "v": 1045
-                    }
-                };
+
+        _getLowerResultLatestIndicator(lowerResultId) {
```
Comment (daniel-tabarcea) [inline pmp/app-elements/reports/report-progress.html]: _getLowerResultLatestIndicator: function(lowerResultId)

Comment (adi130987) [reply pmp/app-elements/reports/report-progress.html]: done :)

**Comment by adi130987** [comment]
TODO: update etools-ajax with abort request functionality

**Pull request merged**